api/utils.go

```diff
@@ -22,7 +22,9 @@ func RunAsync(f func() error) <-chan error {
 
 // HashMessage calculates the hash of a message to be safely signed by the keycard
 // The hash is calulcated as
-//   keccak256("\x19Ethereum Signed Message:\n"${message length}${message}).
+//
+//	keccak256("\x19Ethereum Signed Message:\n"${message length}${message}).
+//
```

Comment:
is this change necessary?

Reply:
i agree, `make lint-fix` did these things, do u have this issue? @rasom